src/styles/utils/_variables.scss

```
@@ -0,0 +1,13 @@
+$days-of-week:
+  mon 0,
+  tue 101px,
```

Better to show that those are just strings.
```suggestion
  "mon" 0,
  "tue" 101px,
```